backend/fms_core/services/sample.py

```diff
@@ -149,6 +149,7 @@ def inherit_sample(sample_source, new_sample_data, derived_samples_destination,
                                            volume_ratio=volume_ratios[derived_sample_destination.id])
         
         # project inheritance
+        # TODO: Move project to derived?
```

Comment:
I actually patched that in my PR so it does not try to create multiple association for the same project. I think we'll be moving the project to Derived_sample. In that situation, this part will become obsolete and we'll remove it in the project migration PR.